scene/2d/light_occluder_2d.cpp

```diff
@@ -237,12 +237,8 @@ Ref<OccluderPolygon2D> LightOccluder2D::get_occluder_polygon() const {
 }
 
 void LightOccluder2D::set_occluder_light_mask(int p_mask) {
-	mask = p_mask;
-	RS::get_singleton()->canvas_light_occluder_set_light_mask(occluder, mask);
-}
-
-int LightOccluder2D::get_occluder_light_mask() const {
-	return mask;
+	set_light_mask(p_mask);
```

Comment:
This is wrong IMO, you're now calling
```
        RS::get_singleton()->canvas_item_set_light_mask(canvas_item, p_light_mask);
```
which did not happen previously.

Those two properties have the same name but they're *not* the same, at least in the current API.